projects_portfolio/styles.css

```diff
@@ -0,0 +1,522 @@
+:root {
+  --light-yellow: #f5e8b9b2;
+  --purpple: #b597f4;
+  --purple2: #ab87f6;
+  --pink: #f7aec7;
+  --white: #fff;
+  --blue: #c1eafc;
+  --dark_blue: #77cbef;
+}
+@font-face {
+  font-family: "Sniglet";
+  src: url(Sniglet/Sniglet-Regular.ttf);
+}
+
+body {
```

Comment:
When you inspect the page in netlify, your `body` tag has 8 pixels. To remove it you can add : margin: 0;
<img width="1211" alt="Screenshot 2023-10-31 at 14 57 01" src="https://github.com/HackYourFuture-CPH/class27-homework/assets/93525817/648050bc-494d-44b6-8519-61b22c8bfe88">